### ISSUE TITLE
Feed re-design recomendations

### ISSUE THREAD

**Issue by Tratcher** [seamonkeysocial/twitarr#9]
For a number of reason the design of the main tweet feed should be revisited:
- Overall usage keeps increasing, there's too much content for the current format
- This year it got really noisy with so many pokemon sightings.
- There were issues with sea mail so individuals were communicating with their parties via the main feed
- There were multiple conversations with content that many people didn't want to watch.

Proposals:
- Discontinue the universal feed, i.e. Users don't follow all other users by default.
- Prefer forums for topical discussions.
  - Forum threads are easier to scroll past if you're not interested in the topic (Pokemon, librarians, kink), and less likely to inflame if users must click-through / subscribe to see the content (?).
  - Allow a user to hide/ignore a forum thread for piece of mind and to improve discovery other threads (active threads are always on top right now).
  - Pre-seed popular topics. E.g. Pokemon, food recommendations, main events (created automatically prior to the event), etc..
- Allow building a personal feed (like the real Twitter)
  - Follow an individual's public tweets
  - Refer users with empty feeds to the forums.
  - Follow forum threads
  - Follow hashtags
  - Follow sea-mail conversations
- Show a user's tweets (but not forum posts) on their profile so you can decided if you want to follow them

Overall, having an opt-in content model should help reduce the noise and may also help reduce the reliance on [moderation ](https://github.com/seamonkeysocial/twitarr/wiki/Moderation-Suggestions) features (e.g. bocking, reporting).

**Comment by chicazul** [comment]
The only issue I have with discontinuing the universal feed is discoverability—how do you find people to follow? A week is not long enough to organically grow a feed. I think dropping the universal feed entirely will lead to a large drop in usage. Which may be desired?

That said, I found many of the sane problems with noise this year and wished I could create a curated feed, with the universal feed as a backup I can look at if I choose instead of the default screen.

**Comment by Tratcher** [comment]
Yes discovery will be harder but I expect it to still be possible via a few different network effects:
- Discover users in person or via direct recommendation
- Discover users from forums
- Discover users and hashtags from other users mentions

**Comment by grundoon** [comment]
Strongly opposed to nuking the main stream or even making it non-default.
The overriding non-starter on that for me: who will se a cry for help, a request for support?

**Comment by Tratcher** [comment]
Who can see it now in the flood?

**Comment by RangerRick** [comment]
Still worth having for discovery purposes.  I do think we should probably have a secondary feed of only people you follow/favorite. (twitarr already has user-favoriting, just not in the feed AFAIK)

**Comment by RangerRick** [comment]
Also, blocking/muting functionality will make a difference for getting rid of "noise" too.  Ideally we could also mute words or hashtags.

**Comment by grundoon** [comment]
Everybody, in theory. A person in crisis should not have to find "the right place". Twitarr serves all; filters serve the individual.

**Comment by grundoon** [comment]
Also @Tratcher, please elaborate on "issues with sea mail". (There were no problem reports submitted of which I am aware.)

**Comment by Tratcher** [comment]
Users on the Android client were unable to initiate sea mail conversations, they could only be initiated via the web app. Once imitated everything worked normally. That's a topic for another thread, I was only mentioning it as a source of noise.

**Comment by chicazul** [comment]
I think the universal feed is one of the main ways newer/shyer attendees can form connections and serendipitous plans can form, but I am weighing that against the people who said we valued our toy more than their ability to feel safe.

If we added a feed of favourites and the ability to mute keywords, twitarr would be *my* ideal social media platform. Based on our discussed moderation options I am tentatively optimistic about making everyone feel secure using the main feed.

**Comment by RangerRick** [comment]
The android client bug was a CruiseMonkey one, not a Twitarr one, so doesn't really apply here.

**Comment by joey** [comment]
I agree that we should keep the "universal feed". Specifically regarding the someone in crisis looking for help, I saw such a message and was able to help someone through a tough time.

Also, I agree with everything @chicazul wrote. Putting all of those features together would be my ideal social media platform as well.

**Comment by hendusoone** [comment]
I also think the main feed should stay. Extending the "favorite this user" functionality into a second personally-curated feed is a great idea too - maybe we should change it from "favorite" to "follow," though.

Muting keywords/hashtags would go a long way to cleaning up the noise in the main feed. Would it be worth it to have a setting to temporarily shut off all your mutes, for the times you want the firehose?

**Comment by hendricksond** [comment]
Looking through this again, and after working on Twitarr for a bit, a curated feed of starred users is already available and working. See Issue #107 , the goal is to make this easier to access before the cruise.
